packages/maskbook/src/extension/background-script/EthereumServices/request.ts

```diff
@@ -0,0 +1,37 @@
+import type { RequestArguments } from 'web3-core'
+import type { JsonRpcPayload, JsonRpcResponse } from 'web3-core-helpers'
+import { INTERNAL_send } from './send'
+
+let id = 0
```

Comment:
This is not a problem in the production, but you should handle HMRs otherwise it might be in a strange state when this service is being HMRed.